core/src/button/flat.module.css

```diff
@@ -5,7 +5,18 @@
 	border: solid 1px transparent;
 }
 
+/* HIGHLIGHT */
+
+:global(.light) .highlight {
+	color: var(--highlight-7);
```

Comment:
@lqt93 Sorry this was my mistake, it should be "highlight-6", not 7! Please help me change it :D